### DISCUSSION TITLE
UI Testing and improving test coverage

### DISCUSSION THREAD

**Discussion by rvema** [FannieMaeOpenSource/metrostop#135] [Ideas]
Starting this discussion on what is the best way to perform UI testing on metrostop so that the project is thoroughly tested with every pull request raised ensuring utmost quality of this flagship project.

**Comment by rvema** [comment]
tagging @andreology and @skybrodowski  for thoughts

**Comment by skybrodowski** [comment]
I thought it would be beneficial to add in unit tests to ensure data is properly transformed and elements are properly created or removed. To that end, I added in new unit tests in Jest to this branch to testscomponents related to the first level of the metrostop. I thought it would be beneficial to work on testing in phases and focus on each level of the metrostop in each phase. Would it make sense to add a Jest command into our GitHub actions file?
